src/mpid/ch4/netmod/ofi/ofi_am_impl.h

```diff
@@ -136,46 +135,21 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_repost_buffer(void *buf, MPIR_Request * r
     goto fn_exit;
 }
 
-MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_progress_do_queue(int vni_idx)
+MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_progress_poll(int vni)
 {
-    int mpi_errno = MPI_SUCCESS, ret;
-    struct fi_cq_tagged_entry cq_entry;
-
-    /* Caller must hold MPIDI_OFI_THREAD_FI_MUTEX */
-
-    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_OFI_PROGRESS_DO_QUEUE);
-    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_OFI_PROGRESS_DO_QUEUE);
-
-    ret = fi_cq_read(MPIDI_OFI_global.ctx[vni_idx].cq, &cq_entry, 1);
```

Comment:
If you're removing the `fi_cq_read` here, where do we call it instead?

Reply:
The regular progress will process all the cq entries.

Reply:
Ok. I guess I missed that it was still there.